terraform/cross-device/storage.tf

```diff
@@ -51,3 +51,7 @@ module "buckets" {
     }
   }]
 }
+
+resource "random_id" "bucket_prefix" {
```

Comment:
You don't need this, see my other comment above.

Reply:
Done